docs/cloud/features/scheduler/scheduler.md

```diff
@@ -165,20 +165,40 @@ This gives you complete control over data security and network access while stil
 
+### Deployment Options
+
+You can deploy the executor containers using any method that works for your infrastructure and operational requirements. The executors are standard Docker containers that can be deployed in any container environment as long as they're properly configured with the required environment variables.
+
+For your convenience, we provide two reference implementations:
+
+1. **Kubernetes with Helm Chart**: For production environments, we provide a [Helm chart](./scheduler/hybrid-executors/README.md) that includes robust configurability, secret management, and scaling options.
+
+2. **Docker Compose**: For simpler environments or testing, we offer a [Docker Compose setup](./scheduler/hybrid-executors-docker-compose.md) to quickly deploy executors on any machine with Docker.
+
+You're free to adapt these reference implementations or create your own deployment method that fits your specific needs. The only requirement is that two executor instances must be running (one for run operations and one for apply operations) and properly configured.
+
 ### Configuration
 
-Exact configuration is left to the user and will vary based on the infrastructure and setup of the user, for example the executors could be run on a Kubernetes cluster or as a standalone pair of Docker containers depending on the user's infrastructure. The following details are an example of how this is done for a Postgres data warehouse and a pair of local containers running in docker.
+The following covers basic configuration concepts for hybrid executors. For detailed configuration options, please refer to the specific documentation for your chosen deployment method above.
+
+Tobiko Cloud requires 2 executor instances to be running:
 
-Tobiko Cloud requires 2 docker instances to be running, one to pick up runs and one for plan applications. The entrypoint for both is `executor run` and `executor apply` respectively. The executor container can be found on [Docker Hub](https://hub.docker.com/r/tobikodata/tcloud). In addition to running the container, the user will need to configure the executor with environment variables that point to Tobiko Cloud as well as the data warehouse.
+1. **Run Executor**: Handles scheduled model execution
+2. **Apply Executor**: Handles applying changes to the data warehouse
 
-To connect to the Tobiko Cloud, the user will need to provide the following environment variables, replaced with the user's own values.
+Both executors need to be properly configured with environment variables to connect to Tobiko Cloud and your data warehouse.
+
+#### Basic Environment Variables
+
+To connect to Tobiko Cloud, you'll need to provide:
 
 ```env
 TCLOUD_URL=https://cloud.tobikodata.com/sqlmesh/acme/analytics_project
-TCLOUD_TOKEN=your_token
+TCLOUD_CLIENT_ID=your_client_id
+TCLOUD_CLIENT_SECRET=your_client_secret
```

Comment:
add an image like this.

> You can create a client secret by navigating your Tobiko Cloud account like the below.

![image](https://github.com/user-attachments/assets/68353861-f325-4314-9946-0c44a8396285)